Sources/Commands/SwiftPackageTool.swift

```diff
@@ -896,6 +899,226 @@ extension SwiftPackageTool {
         }
     }
 
+    struct InstallPackage: SwiftCommand {
+        static let configuration: CommandConfiguration = CommandConfiguration(
+            abstract: "Clone, compile, and copy a remote executable target"
```

Comment:
I don't think it makes sense to install targets. A target is "private" in a package, while a product is what's actually exposed and is public. @neonichu WDYT? 
```suggestion
            abstract: "Clone, compile, and install an executable product."
```